dockerfiles/Dockerfile.marble-injector

```diff
@@ -0,0 +1,16 @@
+FROM alpine/git:latest AS pull
+RUN git clone -b feat/webhook https://github.com/edgelesssys/marblerun.git /marble-injector
+WORKDIR /marble-injector
+
+FROM ghcr.io/edgelesssys/edgelessrt-dev AS build
+COPY --from=pull /marble-injector /marble-injector
+WORKDIR /marble-injector/build
+RUN cmake ..
+RUN make marble-injector
+
+FROM scratch AS release
+LABEL description="EdgelessMarbleInjector"
+COPY --from=build /lib/x86_64-linux-gnu/* /lib/x86_64-linux-gnu/
```

Comment:
Compiling with CGO_ENABLED=0 works but will break automatic unit testing.
We could try changing the tests or just copy the needed libraries into the docker container to avoid the issue

Reply:
replaced the ci image, CGO_ENABLED=0 should work now